### PR TITLE
fix(client): Collapsing Reflex Containers.

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Set reflex-container height to that of the desktop-layout class, or height: calc(100vh - var(--header-height) - 42px). The height needs to be set to prevent the code editor, console, and preview from collapsing. This is only a problem on the freecodecamp.rocks beta site now, and may have been caused by the new alert above the header with the link to report bugs.

Here is a picture of the current problem from the Gitter chat:

image

cc @Bouncey, @raisedadead, @ValeraS, @RandellDawson

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: client/src/templates/Challenges/classic/DesktopLayout.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class DesktopLayout extends Component {
           <ReflexSplitter propagate={true} {...resizeProps} />
           <ReflexElement flex={1} {...resizeProps}>
             {challengeFile && (
-              <ReflexContainer key={challengeFile.key} orientation='horizontal'>
+              <ReflexContainer className='desktop-layout' key={challengeFile.key} orientation='horizontal'>
```
Comment (ValeraS) [inline client/src/templates/Challenges/classic/DesktopLayout.js]: @scissorsneedfoodtoo Let's do `desktop-layout` `id` on the outer `ReflexContainer`. What do you think?

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: client/src/templates/Challenges/classic/DesktopLayout.js (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply client/src/templates/Challenges/classic/DesktopLayout.js]: Yes, that's a much better idea! Thank you. Making the change now

Comment (scissorsneedfoodtoo) [reply client/src/templates/Challenges/classic/DesktopLayout.js]: @ValeraS, I may have spoken too soon. While it seems like the `ReflexSplitter` component can take an `id`, `ReflexContainer` and `ReflexElement` cannot.

What do you think is best? We could stick with the class names for both `ReflexContaners` or we could wrap everything within the `Fragment` tags in a `div` and set its id or class name to `desktop-layout`. Or if you have another idea I'd be happy to test and implement that here

Comment (ValeraS) [reply client/src/templates/Challenges/classic/DesktopLayout.js]: Let's use `#learn-app-wrapper .desktop-layout` selector.

Comment (scissorsneedfoodtoo) [reply client/src/templates/Challenges/classic/DesktopLayout.js]: Alright, sounds good. I'll go ahead and do that now

**Review by ValeraS — APPROVED** [review]

**Pull request merged**